PRIVACY-POLICY.md

```diff
@@ -0,0 +1,21 @@
+<h1 align=center> PRIVACY POLICY </h1>
+
+This privacy policy describes what, how, and why *AntAlmanac* and the technologies that it implements collect, store, and utilize the information that you or your device shares through your use of the website. We take reasonable precautions to protect your data, but you should NOT enter sensitive information into *AntAlmanac*. We may change this policy when necessary. By using *AntAlmanac*, you consent to this privacy statement.
```

Comment:
```suggestion
This privacy policy describes *AntAlmanac*'s practices regarding user data collected from website activity,
including collection and distribution via third party integrations, like *Google Analytics*.
We take reasonable precautions to protect user data, 
but users are responsible for keeping their own sensitive information secure. 
We may change this policy when necessary.
By using *AntAlmanac*, you consent to this privacy statement.
```

Reply:
The policy is fundamentally a contract between us (or _AntAlamanac_) and individual users that they agree to as a condition to use the website. It's not just a statement, so we shouldn't use the second person. Also, if we collect sensitive or personally identifiable information, we are responsible; if we don't, then it doesn't concern us, so we can't say that users are categorically responsible for it.

I like the restructuring though—I write excessively long sentences (as demonstrated above). I'll incorporate that.